doc/mgr/influx.rst

```diff
@@ -0,0 +1,192 @@
+=============
+Influx Plugin 
+=============
+
+The influx plugin continuously collects and sends time series data to an influxdb database. 
+Users have the option to specify what type of stats they want to collect. Some default counters are already set. 
+However, users will have the option to choose some additional counters to collect 
+
+-------------
+Configuration 
+-------------
+
+In order for this module to work, a configure file must be created ``/etc/ceph/influx.conf``
+
+^^^^^^^^
+Required 
+^^^^^^^^
+
+The configurations must include the following under the header ``[influx]``
+
+:Configuration: **Description**
+:interval: Sets how often the module will collect the stats and send it to influx
+:hostname: Influx host
+:username: Influx username
+:password: Influx password
+:database: Influx data base, if a database doesn't alread exist in influx the module will create one 
+:port: Influx port 
+:stats: Stats about the osd, pool, and cluster can be collected. Specify as many as you would like, but seperate each type by a comma
+
+
+^^^^^^^^
+Optional 
+^^^^^^^^
+
+Users have the ability to collect additional counters about both per osd or per cluster under the the header ``[extended]``.
+More information on the exteneded option can be found below under the *extended* section. Seperate each additional congifurations with a comma.  
+
+Example config file:
+
+::
+    [influx]
+        interval = 10
+        hostname = samplehost
+        username = admin
+        password = pass 
+        database = default 
+        port = 8086 
+        stats = osd, pool, cluster
+
+    [extended]
+        osd = op_latency, recovery_ops
+        cluster = op_latency
+
+--------
+Enabling 
+--------
+
+To enable the module the following steps must be taken in order:
+- Load the module by including this in the ceph.conf file 
+
+::
+
+    [mgr]
+        mgr_modules = influx 
+
+- Restart the mgr-daemon ``systemctl restart ceph-mgr@<mymonitor>`` 
+- Initialize the module to run every set interval  ``ceph mgr module enable influx``
```

Comment:
This process has also changed.  You can simply
<pre>
ceph mgr module enable influx
</pre>
to enable and the mgr daemons will restart on their own (as necessary)